src/components/EventChat/CommentCard.styles.ts

```diff
@@ -14,9 +14,5 @@ export const commentCardStyles = makeStyles(() => {
     listIcon: {
       minWidth: '40px',
     },
-    input: {
-      width: 'calc(100% - 10px)',
-      padding: '0 5px',
-    },
```

Comment:
Removed because we don't need padding if we use the filled variant of `TextField`